docs/Tutorials/Migrating-to-Tessera.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Migrating to Tessera
+description: Migrate your Orion configuration and data to Tessera.
+---
+
+# Migrating to Tessera
+
+## Migration Process
+
+A utility is included in Tessera which enables migration of an Orion config
+file and database to a Tessera config file and database.
+
+A full migration workflow would be as follows:
+
+1. Download or build the migration utility
+1. Shut down the Orion and Hyperledger Besu nodes
+1. Perform the config and database migration
+1. Start Tessera and Hyperledger Besu nodes with the new config and database files
+
+## Build Migration Utility
+
+The utility can be built from the [Tessera repository](https://github.com/ConsenSys/tessera/)
+
+1. First clone the Tessera repostory:
+
+`git clone https://github.com/ConsenSys/tessera`
+
+1. Navigate to the orion-to-tessera migration utility directory:
+
+`cd migration/orion-to-tessera/`
+
+1. Build the migration utility using gradle:
+
+`gradle clean build`
```

Comment:
did this work for you? I was still building it from the tessera dir

Reply:
It did work for me at one point - and this follows the instructions on the commit, but after all my problems building on the M1 arch I've asked @NicolasMassart to help test!